src/components/ReportActionItem/IOUPreview.js

```diff
@@ -146,9 +146,9 @@ const IOUPreview = (props) => {
     const moneyRequestAction = ReportUtils.getMoneyRequestAction(props.action);
 
     // If props.action is undefined then we are displaying within IOUDetailsModal and should use the full report amount
-    const requestAmount = props.isIOUAction ? moneyRequestAction.total : ReportUtils.getMoneyRequestTotal(props.iouReport);
+    const requestAmount = props.isIOUAction ? moneyRequestAction.amount : ReportUtils.getMoneyRequestTotal(props.iouReport);
     const requestCurrency = props.isIOUAction ? moneyRequestAction.currency : props.iouReport.currency;
-    const requestComment = Str.htmlDecode(moneyRequestAction.comment).trim()
+    const requestComment = Str.htmlDecode(moneyRequestAction.comment).trim();
```

Comment:
This was missed by linter before

Reply:
Could we fix in a different PR so the review is not preventing it from being fixed?

Reply:
https://github.com/Expensify/App/pull/18985